test/DropdownButton.spec.js

```diff
@@ -12,12 +12,12 @@ describe('<Dropdown />', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
-  test('does not update ID reference with each render', () => {
-    wrapper = shallow(<Dropdown trigger={<span>hi</span>} />);
-    const { idx } = wrapper.instance();
-    wrapper.render();
-    expect(wrapper.instance().idx).toEqual(idx);
-  });
+  // test('does not update ID reference with each render', () => {
+  //   wrapper = shallow(<Dropdown trigger={<span>hi</span>} />);
+  //   const { id } = wrapper.instance();
+  //   wrapper.render();
+  //   expect(wrapper.instance().id).toEqual(id);
+  // });
```

Comment:
no need for this test as we've handled this by moving idgen to default props

Reply:
Yep, you are right! :smiley: